README.md

```diff
@@ -196,16 +196,21 @@ All paths below are prefixed with `solution.` for fully-qualified naming and are
 
 This section contains the CORTX and third-party authentication information used to deploy CORTX on Kubernetes.
 
+A Kubernetes Secret is used to hold the various passwords and secret keys needed by the various components.
+  - If the `secrets.name` field is specified, then CORTX will create and populate this secrets object, using this specified name.  CORTX will randomly generate any fields that are not mentioned or do not have values specified.
+  - If the `secrets.name_ref` field is specified, then CORTX will expect a Secrets object to already exist with the specified name, which contains the passwords for these fields.  This allows an admin to specify passwords outside of solution.yaml
+
```

Comment:
Can we add an example in a sub-bullet here for how users could create a secret with those specific fields?

Something similar to
```bash
kubectl create secret generic my-cortx-secret \
  --from-literal=kafka_admin_secret=xyz123 \
  --from-literal=consul_admin_secret=abc456 \
  ...
```

Reply:
Good suggestion. I will do that.

Reply:
Done.